src/store/default-state.js

```diff
@@ -0,0 +1,105 @@
+export const defaultRecomendations = [
+  {
+    title: 'Recommentations',
+    id: 'recommentations',
+    items: [
+      {
+        id: 'remediations-default-1',
+        icon: 'play',
+        state: 'success',
+        title: 'Try Streams for Apache Kafka.',
+        action: {
+          title: 'Get Started',
+          href: 'INSERT URL HERE',
+        },
+      },
+      {
+        id: 'remediations-default-2',
+        icon: 'play',
+        state: 'success',
+        title: 'Try API management',
+        action: {
+          title: 'Get Started',
+          href:
+            'https://www.redhat.com/en/technologies/cloud-computing/openshift/openshift-api-management',
+        },
+      },
+    ],
+  },
+  {
+    title: 'Insights for Ansible',
+    id: 'ansible',
+    items: [
+      {
+        id: 'ansible-default-1',
+        icon: 'play',
+        state: 'success',
+        title:
+          'Red Hat Ansible Automation Platform provides analytics and knowledge on your automation, access to certified content, and more.',
+        action: {
+          title: 'Try it',
+          href:
+            'https://www.redhat.com/en/technologies/management/ansible/try-it',
+        },
+      },
+    ],
+  },
+  {
+    title: 'Insights for OpenShift',
+    id: 'openshift',
+    items: [
+      {
+        id: 'openshift-default-1',
+        icon: 'play',
+        state: 'success',
+        title:
+          'Your account team can set up Subscriptions to report on your Red Hat OpenShift Container subscription usage.',
+        action: {
+          title: 'Contact Us',
+          href: 'https://access.redhat.com/account-team',
+        },
+      },
+      {
+        id: 'openshift-default-2',
+        icon: 'play',
+        state: 'success',
+        title:
+          'Red Hat OpenShift customers have access to Cost Management, which provides visibility and analysis for your OpenShift and cloud costs.',
+        action: {
+          title: 'Learn More',
+          href:
+            'https://www.redhat.com/en/technologies/cloud-computing/openshift',
+        },
+      },
+    ],
+  },
+  {
+    title: 'Insights for RHEL',
```

Comment:
@PanSpagetka This should be fully spelled out as of Friday afternoon: "Insights for Red Hat Enterprise Linux"